### ISSUE TITLE
Regression: In version 3.3.0, some window control IDs, existing in version 3.2.5, have been removed. For what ?

### ISSUE THREAD

**Issue by paulber19** [audacity/audacity#4988]
### Bug description

Hi.
I noticed the removal of some window control ID since version 3.0. in particular, the control IDs of objects:
- audio position timer (2801)
- start timer selection (2705)
- selection duration timer (2706)
- selection center timer (2707)
- select end timer (2708)
These control IDs are not useless and are essential to correctly identify objects and make Audacity accessible to blind people.
For information, I am the developer of the "audacityAccessEnhancement" add-on for NVAccess's NVDA (Non Visual Desktop) screen reader.
Since version 3.3.0, it is no longer possible to recognize the objects mentioned above in order to announce the audio position, the start or the end of the selection without leaving the track table. This avoids keyboard manipulation for blind people.
I therefore ask the developers of Audacity to return as soon as possible all the control IDs which have been removed and which were in version 3.2.5
I will go even further, it is essential to keep in mind at all times during the development of software, its accessibility by blind people, i.e.:
- all the functions of the software must be achievable using the keyboard.
- each object must be clearly identifiable by a screen reader (for example its window class, its window control ID)
Best regards.
Paul.

### Steps to reproduce

ras

### Expected behavior

I therefore ask the developers of Audacity to return as soon as possible all the control IDs which have been removed and which were in version 3.2.5

### Actual behavior

Some window control IDs has been removed

### Audacity Version

latest stable version (from audacityteam.org/download)

### Operating system

Windows

### Additional context

None

**Comment by DavidBailes** [comment]
> I noticed the removal of some window control ID since version 3.0. in particular, the control IDs of objects:

As far as I can see, the id for the audio position is still 2801.
Unfortunately the ids for the times in the selection toolbar were not replicated from previous versions.
I've just submitted a pull request to put them back: https://github.com/audacity/audacity/pull/5006

Note that using version 2.9 of your NVDA addon, it doesn't seem to read the audio position, and it doesn't read the selection times using my pull request. Not sure what the problem is, but I think the Window ids are all present in the pull request.

**Comment by paulber19** [comment]
Hi David.
Thank you for taking my request into account and especially for your 
investment in making Audacity accessible to the blind.
Indeed, I was wrong for the audio position. The id is present.
Are you using the add-on with Audacity version 3.3.3?
Version 2.9 of the add-on is not compatible with Audacity versions 
3.3.x. It stops at 3.2.5.
I am preparing version 3.0 of the add-on which will be.
Best regards.
Paul.

**Comment by DavidBailes** [comment]
Hi Paul,
Using version 2.8 of your addon and version 3.2.5 of Audacity, I can read both the audio position and the selection.
But using version 2.9 of your addon and version 3.3.3 of Audacity, I just get silence when trying to read the audio position.
David.

**Comment by chinakov** [comment]
@DavidBailes  - it seems like the 2.9 version of the addon is not compatible with 3.3.3 version of Audacity, as per @paulber19 "Version 2.9 of the add-on is not compatible with Audacity versions
3.3.x. It stops at 3.2.5."

So, it seems to me that this ticket can be closed, as window ID's are back, and the compatibility between NVDA and Audacity will be fixed in version 3.0 of NVDA, correct? 

If so, I will close this task as nothing was broken.

**Comment by LWinterberg** [comment]
Window IDs will be back if https://github.com/audacity/audacity/pull/5006 gets merged, otherwise not

**Comment by DavidBailes** [comment]
As @LWinterberg has said, this issue will only be fixed when https://github.com/audacity/audacity/pull/5006 is merged. It's currently still in QA. There are no fixes needed for this issue in the NVDA addon.

**Comment by chinakov** [comment]
Tested, seems nothing was broken by #5006. Thanks!
